content/vendor/guides/ha-guide.md

```diff
@@ -0,0 +1,411 @@
+
+```shell
+sudo mv /etc/haproxy/haproxy.cfg /etc/haproxy/haproxy.bak
+sudo mv ~/haproxy.cfg /etc/haproxy/haproxy.cfg
+```
+
+Last step is to restart Ha Proxy
+
+```shell
+sudo systemctl restart haproxy
+```
+
+To validate the install, browse to http://<public-ip-of-vm>:8080. Below is a screenshot of what it should look like. The dashboard at this point does not contain much valuable information as all the frontend/backend services we configured are not running yet.
+
+![haproxy](/images/guides/kots/ha-cluster-ha-proxy.png)
+
+### Run the Kots Installer & Deploy Application
+
+The next step is to install Kots on the first node, in our case, app-direct-node-01. It's important to note that in reality, you could run the installer on any of the three nodes.
+
+To run the installer, ssh into the node and run the 'one line installer'. You can get this command from the Vendor portal as shown below.
+
+![Channels](/images/guides/kots/ha-cluster-portal-channels.png)
+
+However, in order to enable High Availability we are going to add the '-s ha' [advanced option](https://kurl.sh/docs/install-with-kurl/advanced-options)
+
+```shell
+curl -sSL https://k8s.kurl.sh/appdirect-unstable | sudo bash -s ha
+```
+
+When you include the '-s ha' option, you will be promoted right away for the load balancer address. As shown in the figure below, we are providing the 'internal' IP address. This address is used by the Kubernetes services in the nodes to talk to each other. Using a public IP addresses addds extra layers that are not needed. 
+
+This process will take several minutes as it will install Kubernetes as well as the Application Administration toolset. Once it is finished, you should see something similar as shown below:
+
+![Output](/images/guides/kots/ha-cluster-install-output.png)
+
+Highlighted in red in the screenshot above are first the Kots Admin URL and password to login. Also highlighted are the commands to run on other nodes to join the cluster. In this exercise we are not going to run these commands yet and instead continue deploying the application. The commands to join the cluster are also available in the Kots Admin UI, which we will cover later in this guide.
+
+Next, we are going to log in to Kots Admin to complete the install and deploy the application. Since this guide assumes you have a level of familiarity with Kots, we are not going to go into detail on every step to get the application deployed.
+
+![Log-In](/images/guides/kots/ha-cluster-log-in.png)
+
+Log in to the Aplication Administration Console using the address and password from the install. Once you have logged in using the provided password, you will need to provide a license. The license is generated in the Vendor Portal and this guide assumes you already know how to do this. There are no options that need to be enabled or turned on in order to support an HA cluster, so if you already have a devlopment license, it should work for this exercise.
```

Comment:
I think since we just mentioned above this section that "you should already know how to do this", we might be able to omit it here. What do you think?

Also - small typo here 
```suggestion
Log in to the Aplication Administration Console using the address and password from the install. Once you have logged in using the provided password, you will need to provide a license. The license is generated in the Vendor Portal and this guide assumes you already know how to do this. There are no options that need to be enabled or turned on in order to support an HA cluster, so if you already have a development license, it should work for this exercise.
```